modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -308,7 +318,11 @@ The OTLP gRPC exporter exports data using the OpenTelemetry protocol (OTLP).
 ----
 <1> The OTLP gRPC endpoint. If the `+https://+` scheme is used, then client transport security is enabled and overrides the `insecure` setting in the `tls`.
 <2> The client side TLS configuration. Defines paths to TLS certificates.
-<3> Headers are sent for every RPC performed during an established connection.
+<3> Disables client transport security when set to `true`; `false` by default.
+<4> Skips verifying the certificate when set to `true`; `false` by default.
+<5> Specifies the time interval at which the certificate is reloaded. If not set, the certificate is never reloaded. Accepts a string containing valid units of time such as `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
+<6> If set to a non-empty string, it will override the virtual host name of authority (e.g. :authority header field) in requests (typically used for testing).
```

Comment:
I'm confused about `<6>` and `<7>` now. Could you please check these two callouts in the codeblock?
```

Reply:
```suggestion
<6> When set to a non-empty string, overrides the virtual host name of authority such as the authority header field in requests. Use this for testing.
```